README.md

```diff
@@ -1257,6 +1257,70 @@ Writes the given text to the clipboard
 writeTextToClipboard: (text: string) => Promise<void>;
 ```
 
+### showLoadingOverlay / hideLoadingOverlay
```

Comment:
@dpastor, if you have a video with a demo I can add it to the docs

Reply:
We could add this one (It's not yet verified, so maybe something changes until release):

https://github.com/user-attachments/assets/06613a96-28bd-4b26-bd84-ba5e4add4ae9